HelpfulItem.h

```diff
@@ -1,15 +1,12 @@
 #pragma once
 
 #include "Item.h"
-#include "Dwarf.h"
-#include "Character.h"
-#include "Paladin.h"
-#include "DragonSlayer.h"
-#include "Dragon.h"
+// #include "Character.h"
+struct Character;
```

Comment:
This forward declaration is unnecessary.  `Item.h` is already declaring it
- [ ] revise